Sources/Experiment/ExperimentClient.swift

```diff
@@ -677,6 +677,14 @@ internal class DefaultExperimentClient : NSObject, ExperimentClient {
             print("\(formatter.string(from: Date())) [Experiment] \(msg)")
         }
     }
+    
+    private func shouldRetryFetch(_ e: Error) -> Bool {
+        if (e is FetchError) {
+            let fetchError = e as! FetchError
+            return fetchError.statusCode < 400 || fetchError.statusCode >= 500 || fetchError.statusCode == 429
+        }
+        return true
```

Comment:
The more swifty way to implement this logic is to use `if let` or `guard let` statements. Double check this code in the IDE, I just wrote it in this github comment.

```suggestion
        guard let e = e as FetchError else {
            return true
        }
        return fetchError.statusCode < 400 || fetchError.statusCode >= 500 || fetchError.statusCode == 429
```